src/Compilers/CSharp/Portable/CodeGen/Optimizer.cs

```diff
@@ -135,6 +135,8 @@ private static void RemoveIntersectingLocals(Dictionary<LocalSymbol, LocalDefUse
                 }
             }
 
+            var dummyCnt = defs.Count;
```

Comment:
It looks like we can add something to the ```defs``` and ```dummyCnt``` represents its length before we do that. Could we call it ```initialCnt``` instead?